Review this preregistration update (#3)
eviho/repro-collab: preregistration.md

# Replication of Classic Psychophysical Research on Size Discrimination

## Study Overview

Psychophysics is the scientific study of the relationship between physical stimuli (like light, sound, or size) and our psychological perception of them.
A fundamental question in this field is: when something in the world changes by a certain amount, how much does our perception of it change?

> "The tower of Babel was never finished because the workers could not reach an understanding on how they should build it; my psychophysical edifice will stand because the workers will never agree on how to tear it down."  
> — Gustav Fechner (1877)

In this quote, Fechner—the founder of psychophysics—humorously suggests that his framework for understanding how we perceive the physical world will endure precisely because critics cannot agree on a unified approach to challenge it.

> Empierically we can challenge it by trying to replicate an original experiment from the time.

Reviewer: ```suggestion
There is typo, please edit to: **Empirically** we can challenge it by trying to replicate an original experiment.
```
